client/ayon_core/pipeline/colorspace.py

```diff
@@ -879,6 +879,42 @@ def _get_global_config_data(
     }
 
 
+def _is_host_ocio_enabled_n_overridden(host_name, project_settings):
```

Comment:
The `n` for `and` just feels like a typo, either write it in full or just `_get_host_ocio_state` or `_get_host_ocio_settings` or whatever. A docstring ~~can~~ should elaborate what it returns too.

The function name gave me no idea that it returned a tuple - I thought it would return a single `bool` based of the function name on whether it was enabled and had overrides.